1-exercises/B-highlight-words/script.js

```diff
@@ -1,5 +1,21 @@
 function highlightWords(paragraph, colours) {
   // Write your code here...
+  const container = document.getElementById("content");
+  const unorderedList = document.createElement("ul");
+  container.appendChild(unorderedList)
+
+  todos.forEach((todo) => {
```

Comment:
I don't think this is the correct solution for the excercise to highlight words?